src/Sarif/GlobalSuppressions.cs

```diff
@@ -0,0 +1,17 @@
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1703:ResourceStringsShouldBeSpelledCorrectly", MessageId = "cppcheck", Scope = "resource", Target = "Microsoft.CodeAnalysis.Sarif.SdkResources.resources")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1703:ResourceStringsShouldBeSpelledCorrectly", MessageId = "Cpp", Scope = "resource", Target = "Microsoft.CodeAnalysis.Sarif.SdkResources.resources")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif", Scope = "namespace", Target = "Microsoft.CodeAnalysis.Sarif.Converters")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif", Scope = "namespace", Target = "Microsoft.CodeAnalysis.Sarif.Driver")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif", Scope = "namespace", Target = "Microsoft.CodeAnalysis.Sarif.Readers")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif", Scope = "namespace", Target = "Microsoft.CodeAnalysis.Sarif.Writers")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Sarif")]
+// This file is used by Code Analysis to maintain SuppressMessage 
```

Comment:
Oh! I see you already have a global suppression for "Sarif" (the `SuppressMessage` attribute has no `Target` property, so it applies to everything). You should be able to remove the individual suppressions.